rules/aws_cloudtrail_rules/aws_ec2_startup_script_change.yml

```diff
@@ -0,0 +1,71 @@
+AnalysisType: rule
+Description: 'Detects changes to the EC2 instance startup script. The shell script will be executed as root/SYSTEM every time the specific instances are booted up. '
+DisplayName: AWS EC2 Startup Script Change
+Enabled: false
+Filename: aws_ec2_startup_script_change.py
+Reports:
+    MITRE ATT&CK:
+        - T1059
+Severity: High
+Tests:
+    - ExpectedResult: true
+      Log:
+        awsregion: us-east-1
```

Comment:
I think `Tests.[].Log` should  be presented in json format for consistency's sake . I didn't realize that straight yaml formatting was possible